core/src/main/scala/tamer/config/Config.scala

```diff
@@ -13,8 +13,6 @@ import zio.interop.catz._
 import scala.concurrent.duration.FiniteDuration
 
 object Config {
-  final case class Db(driver: NonEmptyString, uri: UriString, username: NonEmptyString, password: Password)
```

Comment:
looks like there's two refinement types left in `config`'s package object that should be moved to the doobie module in the `db` package object: `UriString` and `Password`